crypto/rand/drbg_lib.c

```diff
@@ -501,31 +625,47 @@ void *RAND_DRBG_get_ex_data(const RAND_DRBG *drbg, int idx)
  */
 
 /*
- * Initializes the DRBG with default settings.
- * For global DRBGs a global lock is created with the given name
+ * Allocates a new global DRBG on the secure heap (if enabled) and
+ * initializes it with default settings.
+ * A global lock for the DRBG is created with the given name.
+ *
  * Returns 1 on success, 0 on failure
```

Comment:
It returns a drbg on success and NULL on failure

Reply:
Fixed, here and in the comment of RAND_DRBG_new().